luigi/mock.py

```diff
@@ -135,6 +135,10 @@ def rename(self, *args, **kwargs):
     def path(self):
         return self._fn
 
+    @property
+    def fn(self):
```

Comment:
Instead of using `.fn` can't you just use `.path` and there's less code in total?